internal/pathpolicy/policies.go

```diff
@@ -2,16 +2,17 @@ package pathpolicy
 
 // MountpointPolicies is a set of default mountpoint policies used for filesystem customizations
 var MountpointPolicies = NewPathPolicies(map[string]PathPolicy{
-	"/":     {Exact: true},
-	"/boot": {Exact: true},
-	"/var":  {},
-	"/opt":  {},
-	"/srv":  {},
-	"/usr":  {},
-	"/app":  {},
-	"/data": {},
-	"/home": {},
-	"/tmp":  {},
+	"/":         {Exact: true},
+	"/boot":     {Exact: true},
+	"/boot/efi": {Exact: true},
```

Comment:
This will cause issues.  `/boot/efi` needs to be FAT so we'll need a special case because (I THINK) by default we make custom mountpoints xfs on RHEL.

Reply:
I might be wrong.  Checking.

Reply:
I WAS WRONG

Reply:
We generate the partition table in two passes: first pass resizes existing partitions and doesn't create new ones, to avoid changing filesystem types (I guess we thought of this).
Second pass adds new partitions/mountpoints.